inst/validation/bbr-stories.yaml

```diff
@@ -294,14 +404,19 @@ SUM-S003:
     in a NONMEM control stream, when they are in a specified format.
   ProductRisk: Low
   requirements:
+  - PLB-R001
+  - PLB-R002
+  - PLB-R003
+  - PLB-R004
```

Comment:
Not entirely sure if R001, R002, and R004 belong here. It might make sense to delete these requirements entirely, as they're predominantly helper functions. `Block()` is technically exported, but it does very little and dont think it _needs_ a requirement. These three functions certainly do not deserve a new story, so I would either add them to `SUM-S003` like I did above, or remove them from the requirements. Thoughts?

```yaml
PLB-R001:
  description: build_matrix_indices() parses correctly
  tests:
  - BBR-PLB-001
PLB-R002:
  description: block() parses correctly
  tests:
  - BBR-PLB-002
PLB-R004:
  description: parse_param_comment() called internally
  tests:
  - BBR-PLB-004
```

Reply:
oops I didnt highlight the entire code block. Those requirements were added to this story (currently):

```yaml
SUM-S003:
  name: Parse parameter labels from control stream
  description: As a user, I want to be able to parse parameter names from comments
    in a NONMEM control stream, when they are in a specified format.
  ProductRisk: Low
  requirements:
  - PLB-R001
  - PLB-R002
  - PLB-R003
  - PLB-R004
  - PLB-R005
  - PLB-R006
```

Reply:
Its worth noting that we now have to pick one of those options. Having requirements **not linked** to a story is no longer allowed in the newest version of `mrgvalidate`. (of course you can still have tests without requirements though)

Reply:
I'd be in favor of

 * adding `PLB-R002` to `SUM-S003` would be fine, though I agree deleting `PLB-R002` is okay too
 * deleting `PLB-R001` and `PLB-R004`